core/iwasm/aot/aot_runtime.c

```diff
@@ -956,7 +941,8 @@ execute_start_function(AOTModuleInstance *module_inst)
     u.f(exec_env);
 
     wasm_exec_env_destroy(exec_env);
-    (void)clear_wasi_proc_exit_exception(module_inst);
+    (void)clear_wasi_proc_exit_exception(
+        (WASMModuleInstanceCommon *)module_inst);
```

Comment:
Seems no need to clear wasi proc exit exception here. WASI module doesn't set internal start func index，instead it exports a function named "_start". If here it really call wasi proc exit，we can also let instantiation process failed. @lum1n0us what is your opinion?

Reply:
Agree with that, and updated